app/Providers/RouteServiceProvider.php

```diff
@@ -128,6 +128,11 @@ private function mapWebRoutes()
                 Route::get('flights/search', 'FlightController@search')->name('flights.search');
                 Route::resource('flights', 'FlightController');
 
+                // Subfleets
+                Route::get('subfleets', 'SubFleetController@index')->name('subfleets.index');
```

Comment:
Rename to `SubfleetController` so this matches how it's used elsewhere

Reply:
Done